WBooks/DetailBook/DetailBookViewController.swift

```diff
@@ -0,0 +1,100 @@
+        detailBookView.yearLabel.text = detailBookViewModel.book.year
+        detailBookView.genreLabel.text = detailBookViewModel.book.genre
+        detailBookView.statusLabel.text = detailBookViewModel.book.status
+        
+        detailBookView.rentButton.addTarget(self, action: #selector(rentPress), for: .touchUpInside)
+        detailBookView.addWishlistButton.addTarget(self, action: #selector(addWishlistPress), for: .touchUpInside)
+        
+        detailBookViewModel.getComments { comments in
+            if comments.count > 0 {
+                self.detailBookView.tableView.reloadData()
+            } else {
+                self.detailBookView.commentsView.isHidden = true
+            }
+        }
+    }
+    
+    override func loadView() {
+        view = detailBookView
+    }
+    
+    @objc func rentPress(sender: UIButton){
+        if (detailBookViewModel.validateStatusRent()) {
+            detailBookViewModel.rent(){ error in
+                if error == nil {
+                    self.showAlert(title: NSLocalizedString("TITLE_INFORMATION", comment: ""), message: NSLocalizedString("INFORMATION_SUCCESS", comment: ""), buttonDone: NSLocalizedString("BUTTON_DONE", comment: ""))
+                } else {
+                    self.showAlert(title: NSLocalizedString("TITLE_ERROR", comment: ""), message: NSLocalizedString("ERROR_GENERAL", comment: ""), buttonDone: NSLocalizedString("BUTTON_DONE", comment: ""))
+                }
+            }
+        } else {
+            self.showAlert(title: NSLocalizedString("TITLE_INFORMATION", comment: ""), message: NSLocalizedString("ERROR_UNAVAILABLE", comment: ""), buttonDone: NSLocalizedString("BUTTON_DONE", comment: ""))
+        }
+    }
+    
+    @objc func addWishlistPress(sender: UIButton){
+        detailBookViewModel.addWishList()
+    }
+    
+    func showAlert(title: String, message: String, buttonDone: String) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: buttonDone, style: .default, handler: nil))
+        self.present(alert, animated: true)
+    }
+
+}
+
+extension DetailBookViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return detailBookViewModel.comments.count
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return UITableView.automaticDimension
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+                
+        let cell = tableView.dequeueReusableCell(withIdentifier: CommentsTableViewCell.identifier, for: indexPath) as! CommentsTableViewCell
+        
+        detailBookViewModel.getUserComment(id: detailBookViewModel.comments[indexPath.row].user_id){ user in
```

Comment:
Create a method to get the id and avoid chaining of properties to don't create coupling

Reply:
Done